tests/test_ote_training.py

```diff
@@ -383,8 +381,20 @@ class TestOTETraining:
     PERFORMANCE_RESULTS = None # it is required for e2e system
 
     DEFAULT_NUM_ITERS = 1
+    DEFAULT_BATCH_SIZE = 2
+    SHORT_TEST_PARAMETERS_NAMES_FOR_GENERATING_ID = OrderedDict([
+            ('model_name', 'model'),
+            ('dataset_name', 'dataset'),
+            ('num_training_iters', 'num_iters'),
+            ('batch_size', 'batch'),
+            ('usecase', 'usecase'),
+    ])
```

Comment:
Why can't you use short names from the very beginning?

Reply:
I supposed that just `model` instead of `model_name` or just `dataset` instead of `dataset_name` may cause ambiguity in python code -- as an example, model itself is loaded in another part of the code, only model name is here, the same with dataset that is loaded in another part of code too, only dataset name is used here. 
And `num_iters` is not clear in python code too -- there will be num iters for NNCF compression, etc. 

But when we want to generate string id-s for tests I think that brevity is more important, so I make less precise but more short param names.